listenbrainz/timescale_writer/timescale_writer.py

```diff
@@ -195,7 +195,9 @@ def start(self):
                 break
             except Exception:
                 current_app.logger.error("Error in Timescale Writer:", exc_info=True)
-                time.sleep(3)
+                current_app.logger.error("Sleeping 3 seconds and exiting...", exc_info=True)
```

Comment:
Instead of breaking, you can remove the loop entirely but either is fine for now.